docs/mkdocs.yml

```diff
@@ -2,718 +2,721 @@ site_name: CLI for Microsoft 365
 site_url: https://pnp.github.io/cli-microsoft365/
 nav:
   - Home:
-    - 'index.md'
-    - v4 Upgrade Guidance: 'v4-upgrade-guidance.md'
-    - v3 Upgrade Guidance: 'v3-upgrade-guidance.md'
+      - "index.md"
```

Comment:
Ouch, this should not go in. Let's not replace all single quotes with double quotes and only include changes that are a part of the PR.

Reply:
Hmm not quite sure how that got in, let me check!